django/core/management/utils.py

```diff
@@ -173,3 +174,64 @@ def run_formatters(written_files, black_path=(sentinel := object())):
             [black_path, "--fast", "--", *written_files],
             capture_output=True,
         )
+
+
+SPACE_PREFIX = "    "
+BRANCH_PREFIX = "|   "
+TEE_PREFIX = "|___ "
+ELBOW_PREFIX = "|___ "
+
+
+def get_directory_tree(path, max_depth=10, prefix="", depth=0, **options):
+    """
+    Get the folder structure of a directory and its content,
+    akin to the `tree` command.
+    Adapted from: https://stackoverflow.com/a/59109706
+
+    Example:
+    >>> for line in get_directory_tree(Path.home()):
+    ...     print(line)
+    """
+    path = Path(path)
+    if (supports_color() and not options.get("no_color", False)) or (
+        options.get("force_color", False)
+    ):
+        directory_style = make_style("notice=blue,bold;")
+    else:
+        directory_style = no_style()
+
+    if depth == 0:
+        yield directory_style.NOTICE(path.name)
+
+    if not path.is_dir():
+        raise CommandError("'%s' is not a directory" % (path))
+
+    if depth >= max_depth:
+        return
+
+    dir_contents = list(path.iterdir())
+    # contents each get pointers that are ├── with a final └── :
+
+    for index, subpath in enumerate(sorted(dir_contents)):
+        # Make folders appear bolder in terminal
+        subpath_name = (
+            directory_style.NOTICE(subpath.name) if subpath.is_dir() else subpath.name
+        )
+
+        is_last_element = index == len(dir_contents) - 1
+
+        # The last element in the tree gets a different prefix
+        if is_last_element:
+            yield prefix + ELBOW_PREFIX + subpath_name
+        else:
+            yield prefix + TEE_PREFIX + subpath_name
```

Comment:
```suggestion
         yield prefix + ELBOW_PREFIX + subpath_name
```
I feel like TEE and ELBOW are the same?